app/views/pages/moving-guide/tdy.html.erb

```diff
@@ -1,11 +1,132 @@
-<%- @page_title = 'What to expect… TDY' -%>
+<%- @page_title = 'What to Expect with a TDY Move' -%>
 
 <%= render partial: 'shared/contextual_header', locals: {page_title: @page_title, parent_pages: []} %>
 
 <div class="usa-section main-section">
-  <h1>What to expect… <%= abbr_tag('tdy') %></h1>
+  <h1>What to expect with a <%= abbr_tag('tdy') %> Move</h1>
```

Comment:
Should the "e" in "expect" be capitalized to match `@page_title` (line 1) or vice versa?

Reply:
yes, good catch!